src/components/ui/AnswerBoard.js

```diff
@@ -15,7 +14,29 @@ const Wrapper = styled.div`
 `;
 
 const Header = styled.div`
-    margin-bottom: 16px;
+`;
+
+const ReturnButton = styled.button`
+    margin-left: ${props => props.marginLeft || 0}px;
+    margin-top: ${props => props.marginTop || 0}px;
+    padding-top: 3px;
+    padding-bottom: 3px;
+    padding-left: 10px;
+    padding-right: 10px;
+    margin-bottom: 10px;
+    border-radius: 4px;
+    outline: none;
+    font-family: IBM Plex Sans;
+    font-size: 14px;
+    font-weight: bold;
+    text-align: center;
+    color: #333333;
```

Comment:
It should be background-color: SOMETHING

Reply:
Done!